docs/luigi-client-api.md

```diff
@@ -946,60 +975,60 @@ Stores an item for a specific key.
+*   `value` **[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)** item to store; object must be stringifyable
 
 ##### Examples
 
 ```javascript
 LuigiClient.storageManager().setItem('keyExample','valueExample').then(() => console.log('Value stored'))
 ```
 
-Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;void>** resolves an empty value when the storage operation is over. It will launch an error if storage is not supported, the value cannot be stringified, or if you are using a Luigi reserved key.
+Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)\<void>** resolves an empty value when the storage operation is over. It will launch an error if storage is not supported, the value cannot be stringified, or if you are using a Luigi reserved key.
 
 **Meta**
 
--   **since**: 1.6.0
+*   **since**: 1.6.0
 
 #### getItem
 
 Retrieves an item for a specific key.
 
 ##### Parameters
 
--   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** used to identify the value
+*   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** used to identify the value
 
 ##### Examples
 
 ```javascript
 LuigiClient.storageManager().getItem('keyExample').then((value) => console.log);
 ```
 
-Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)>** resolves an item retrieved from storage. It will launch an error if storage is not supported.
+Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)<[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)>** resolves an item retrieved from storage. It will launch an error if storage is not supported.
 
 **Meta**
 
--   **since**: 1.6.0
+*   **since**: 1.6.0
 
 #### removeItem
 
 Removes an item for a specific key.
 
 ##### Parameters
 
--   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** used to identify the value
+*   `key` **[string](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/String)** used to identify the value
 
 ##### Examples
 
 ```javascript
 LuigiClient.storageManager().removeItem('keyExample').then((value) => console.log(value + ' just removed')
 ```
 
-Returns **[Promise](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Promise)&lt;[Object](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object)>** resolves an item just removed from storage. It will launch an error if storage is not supported or if you are using a Luigi reserved key.
```

Comment:
Looks like another change caused by the linting mentioned in #2212  🤔 But I don't think it causes any issues, right?

Reply:
Okay, I hadn't seen this one change, but it does seem to actually break the links. Re-opening the issue #2212

Reply:
Resolved it by manually updating docu.